dig_zone_path_1:
  -
    x: 1.0
    y: 2.0
    theta: 30.0
    reverse: false
    match_orientation: false
    drive_profile: 0
  -
    x: 3.0
    y: 4.0
    theta: 40.0
    direction: forward
    match_orientation: true
    drive_profile: 0